docs/deployment/netconf/registries.mdx

```diff
@@ -8,20 +8,21 @@ The wasmCloud host supports pulling images for [components](/docs/concepts/compo
 
 ## Allowing Insecure Access to a Registry
 
-By default wasmCloud will pull all images from a registry over HTTPS. If you need to pull images from a registry that does not have a valid certificate, you can allow wasmCloud to pull images from that registry by setting the `WASMCLOUD_OCI_ALLOWED_INSECURE` environment variable. For example:
+By default wasmCloud will pull all images from a registry over HTTPS. If you need to pull images from a registry that does not have a valid certificate, you can allow wasmCloud to pull images from that registry by setting the `ALLOWED_INSECURE` environment variable. For example:
 
 ```shell
-export WASMCLOUD_OCI_ALLOWED_INSECURE='myregistry.example.com'
+export ALLOWED_INSECURE='myregistry.example.com'
 wash up
 ```
+You may also include multiple registries in a comma-separated list.
 
 ## Configuring Access to a Private Registry
 
-In production environments, it's common to pull artifacts from a private registry. wasmCloud hosts can configured to authenticate with a private registry using [host configuration](/docs/reference/host-config). For example:
+In production environments, it's common to pull artifacts from a private registry. wasmCloud hosts can configured to authenticate with a private registry using [host configuration](/docs/cli/wasmcloud). For example, the following environment variables can be used to authenticate to the registry specified with the `OCI_REGISTRY` option:
 
 ```shell
-export WASMCLOUD_OCI_REGISTRY_USER='myuser'
-export WASMCLOUD_OCI_REGISTRY_PASSWORD='myp4$$w0rd'
+export OCI_USER='myuser'
+export OCI_PASSWORD='myp4$$w0rd'
```

Comment:
This also doesn't *seem* to have changed yet?

Reply:
Also note that if we're talking about hte host the ENV var here is actually [`WASMCLOUD_OCI_REGISTRY_USER` it seems](https://github.com/wasmCloud/wasmCloud/blob/main/src/main.rs#L233)